### PR TITLE
Extend default labels

### PR DESCRIPTION
**Pull request by kevinwolfdev**
# What Changed

- Added `tests` label with `🧪 Tests` changelog title
- Added `dependencies` label with `🔩 Dependency Updates` changelog title
- Added `performance` label with `🏎 Performance` changelog title

# Why

Because most open source projects can have missing or weak tests, need dependency updates (this happens more often) or maybe an existing feature needs a performance boost.

<details>
  <summary>📦 Published PR as canary version: <code>under canary scope @auto-canary@9.35.2-canary.1206.15947.0</code></summary>
  <br />
  
  :sparkles: Test out this PR locally via:
  
  ```sh
  npm install @auto-canary/bot-list@9.35.2-canary.1206.15947.0
  npm install @auto-canary/auto@9.35.2-canary.1206.15947.0
  npm install @auto-canary/core@9.35.2-canary.1206.15947.0
  npm install @auto-canary/all-contributors@9.35.2-canary.1206.15947.0
  npm install @auto-canary/brew@9.35.2-canary.1206.15947.0
  npm install @auto-canary/chrome@9.35.2-canary.1206.15947.0
  npm install @auto-canary/cocoapods@9.35.2-canary.1206.15947.0
  npm install @auto-canary/conventional-commits@9.35.2-canary.1206.15947.0
  npm install @auto-canary/crates@9.35.2-canary.1206.15947.0
  npm install @auto-canary/exec@9.35.2-canary.1206.15947.0
  npm install @auto-canary/first-time-contributor@9.35.2-canary.1206.15947.0
  npm install @auto-canary/gem@9.35.2-canary.1206.15947.0
  npm install @auto-canary/gh-pages@9.35.2-canary.1206.15947.0
  npm install @auto-canary/git-tag@9.35.2-canary.1206.15947.0
  npm install @auto-canary/gradle@9.35.2-canary.1206.15947.0
  npm install @auto-canary/jira@9.35.2-canary.1206.15947.0
  npm install @auto-canary/maven@9.35.2-canary.1206.15947.0
  npm install @auto-canary/npm@9.35.2-canary.1206.15947.0
  npm install @auto-canary/omit-commits@9.35.2-canary.1206.15947.0
  npm install @auto-canary/omit-release-notes@9.35.2-canary.1206.15947.0
  npm install @auto-canary/released@9.35.2-canary.1206.15947.0
  npm install @auto-canary/s3@9.35.2-canary.1206.15947.0
  npm install @auto-canary/slack@9.35.2-canary.1206.15947.0
  npm install @auto-canary/twitter@9.35.2-canary.1206.15947.0
  npm install @auto-canary/upload-assets@9.35.2-canary.1206.15947.0
  # or 
  yarn add @auto-canary/bot-list@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/auto@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/core@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/all-contributors@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/brew@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/chrome@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/cocoapods@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/conventional-commits@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/crates@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/exec@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/first-time-contributor@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/gem@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/gh-pages@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/git-tag@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/gradle@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/jira@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/maven@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/npm@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/omit-commits@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/omit-release-notes@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/released@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/s3@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/slack@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/twitter@9.35.2-canary.1206.15947.0
  yarn add @auto-canary/upload-assets@9.35.2-canary.1206.15947.0
  ```
</details>

### CODE REVIEW

File: packages/core/src/release.ts [hunk 1]
```diff
@@ -116,6 +116,24 @@ export const defaultLabels: ILabelDefinition[] = [
...
+    name: "dependencies",
+    changelogTitle: "🔩 Dependency Updates",
+    description: "Update one or more dependencies version",
+    releaseType: "none",
+  },
+  {
+    name: "performance",
+    changelogTitle: "🏎 Performance",
+    description: "Improve performance of an existing feature",
+    releaseType: "none",
```
Comment (hipstersmoothie) [inline packages/core/src/release.ts:135]: This should probably be a `patch` right? You would want a release to go out

Comment (kevinwolfdev) [reply packages/core/src/release.ts]: Ok, changed. Do you think `dependencies` should also be a `patch`?

Comment (hipstersmoothie) [reply packages/core/src/release.ts]: I'd say that kind of depends on the user. I like that it has a `none` release type so a can merge a bunch quickly.

Comment (kevinwolfdev) [reply packages/core/src/release.ts]: Ok, cool. Then, as I see it, all changes are made.

**Comment by hipstersmoothie** [comment]
Seems like a test is failing

**Comment by kevinwolfdev** [comment]
> Seems like a test is failing

Ok. Going to fix that later today.

**Comment by kevinwolfdev** [comment]
@hipstersmoothie I fixed the tests, they execute successfully on my local machine but I still see an error on CircleCI but I can't access the logs.

**Comment by hipstersmoothie** [comment]
I'll finish this one up!

**Pull request merged**